### PR TITLE
fonts & bib style

### PR DESCRIPTION
**Pull request by tg-x**
- fonts: use libertine & inconsolata
- bib style: ACM-Reference-Format

### CODE REVIEW

**Comment by heindel** [comment]
one issue I noticed is the wrong behavior for mathit for xelatex and lualatex: it gives roman fonts for some reason, as seen in the penultimate line / the last display on the first page
image

**Comment by jonaprieto** [comment]
I'm not a fan of this new bib style. It makes it confusing (e.g., in B&W). What exactly is the citation number referring to? Also, references by number make the bookkeeping more laborious when reading the paper. The previous style was okay. 

image

Also, we should definitely test for code. We have many ARTs with code already. I noticed you removed the use of JetBrains monofont. Is there any reason for doing that?

Anyway, so far, I see improvements in this PR. GJ!

**Comment by heindel** [comment]
> I'm not a fan of this new bib style. It makes it confusing (e.g., in B&W). 

Isaac requested a different style. I would use one of those two
- alpha
- apalike

File: art.cls [hunk 1]
```diff
@@ -170,15 +120,58 @@
...
+  % \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
+
+  % \defaultfontfeatures{Scale=MatchLowercase}
+  % \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
+\fi
+
+\RequirePackage[libertine,amsthm]{newtx}
+
+\RequirePackage[varqu]{zi4} % inconsolata
+\ifPDFTeX\else
```
Comment (heindel) [inline art.cls:151]: if we change this line to
```
\ifPDFTeX\RequirePackage[scaled=.825]{GoMono}\else
```
I think we would get something that looks about right if one insists in pdflatex for some reason, but  needs minted listings
image

**Comment by tg-x** [comment]
ok i fixed the monofont scaling and cleaned up the main.tex a bit
for the minted test i'm missing some deps so added a toggle for it

**Comment by tg-x** [comment]
> > I'm not a fan of this new bib style. It makes it confusing (e.g., in B&W).
> 
> Isaac requested a different style. I would use one of those two
>     * alpha
>     * apalike

not sure we can or should agree on it right now, so i made it user defined

**Comment by jonaprieto** [comment]
> Isaac requested a different style. I would use one of those two
> 
> * alpha
> * apalike

`alpha` is fine. Btw, better if we use parenthesis for numbering equations, that's normal.

File: test.tex [hunk 2]
```diff
@@ -71,6 +71,19 @@ \subsection{Display Equations}\label{display-equations}
...
+text and centered horizontally---is produced by the \textbf{equation}
+environment. An unnumbered display equation is produced by the
+\textbf{displaymath} environment.
+
+\paragraph{Further details}
+
+Again, in either environment, you can use any of the symbols and
+structures available in \LaTeX@; this section will just give a couple of
+examples of display equations in context. First, consider
+Equation~\ref{eq-oneequation}, shown as an inline equation above:
```
Comment (jonaprieto) [inline test.tex:86]:
```suggestion
~\eqref{eq-oneequation}, shown as an inline equation above:
```

**Pull request merged**